schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/spi/JasifyEndpoint.java

```diff
@@ -40,6 +48,61 @@
                 ownerName = "Jasify",
                 packagePath = ""))
 public class JasifyEndpoint {
+    abstract static class OrgMemberChecker {
+        protected final Key id;
+        protected OrgMemberChecker(Key id) {
+            this.id = id;
+        }
+        public boolean isOrgMember(long userId) throws EntityNotFoundException {
+            Organization organization = getOrganization();
+            com.jasify.schedule.appengine.model.users.User user = UserServiceFactory.getUserService().get(userId);
```

Comment:
Here also (not so bad) but we could query OrganizationMember where orgId=X and userId=Y and never have to get either user or organization